packages/owl/owl.0.5.0/opam

```diff
@@ -18,12 +18,13 @@ Recently, Owl has implemented algorithmic differentiation which essentially make
 build: [
   [ "dune" "subst" ] {dev}
   [ "dune" "build" "-p" name "-j" jobs ]
-  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+#  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+# Tests incompatible with the opam sandbox
 ]
 
```

Comment:
```suggestion
available: arch != "arm32" & arch != "arm64"
```
The latter may be added at some point (https://github.com/owlbarn/owl/pull/570) but it requires manual intervention for now

Reply:
Added. Thanks!